src/electionguard/key_ceremony_mediator.py

```diff
@@ -92,7 +92,7 @@ def share_announced(
 
         guardian_keys: List[ElectionPublicKey] = []
         for guardian_id in self._get_announced_guardians():
-            if guardian_id is not requesting_guardian_id:
+            if guardian_id != requesting_guardian_id:
```

Comment:
Minor little bugfix in the existing codebase.  `is` and `is not` in Python do a reference comparison, which works fine when a key ceremony happens all-at-once, but it fails to compare string values correctly when strings get reconstituted from a database.

Reply:
nice catch